### PR TITLE
[16] Render the related product list on the HTML page with JavaScript

### PR DESCRIPTION
**Pull request by tomike19**
## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@tomike19 Javascript can not interpret `&#163;` update to use `NGN` or `N`

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@tomike19 use camelCase as have done for other to name the ImageUrl

File: submissions/kolajoTomike/Javascript/app.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+var productList=[
+{
+  productName:"STRAWBERRY ICE-CREAM SANDWICHES",
+  ImageUrl:"../html/img/hand.png",
```
Comment (ToluAkin) [inline submissions/kolajoTomike/Javascript/app.js:4]: @tomike19 I think you should change this key name to be in `camelCase` just like the others.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/kolajoTomike/Javascript/app.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+{
+  productName:"COCONUT MILK AND AVOCADO ICE CREAM",
+  ImageUrl:"../html/img/ice3.png",
+  productPrice:"N900.00",
+},
+];
+ productList.forEach((product)=>{
+    console.log(product.productName + ' with '+ product.ImageUrl + ' is sold for ' + product.productPrice)
+ });
+
```
Comment (adaugochi) [inline submissions/kolajoTomike/Javascript/app.js:24]: Remove spaces from line 25 - 28

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**